### DISCUSSION TITLE
Login Page Logo - Save as Retina size instead of 300px

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cms#9042] [Ideas]
### Description

When you upload a "Login Page Logo," it crunches the logo down to 300px.  Can you tranform it as 600px instead?  And then on the login screen, just have it display the 600px image as 300px wide?  This will help the image not appear blurry.  When you have text in the login logo, it will always seem blurry unless its @ 2x.

Thank you!

**Comment by brandonkelly** [comment]
If you have an SVG version of the logo, you can use that instead, which is infinitely scalable.
